src/generators/randgraphs.jl

```diff
@@ -598,6 +598,30 @@ function random_regular_digraph(n::Integer, k::Integer; dir::Symbol=:out, seed::
     end
 end
 
+@doc_str """
+    random_tournament_digraph(n)
+
+Create a random directed [tournament graph]
+(https://en.wikipedia.org/wiki/Tournament_(graph_theory))
+with `n` vertices.
+
+### Optional Arguments
+- `seed=-1`: set the RNG seed.
+"""
+function random_tournament_digraph(n::Integer; seed::Int=-1)
+
+    rng = getRNG(seed)
+    g = DiGraph(n)
+
+    for i = 1:n
+        for j = i:n
+            rand(rng) > 0.5 ? add_edge!(g, Edge(i, j)) : add_edge!(g, Edge(j, i))
+        end
+    end
```

Comment:
In addition to adding tests, could you simplify this?
```
for i = 1:n, j = i:n
  rand(rng) > 0.5 ? add_edge!(g, Edge(i, j)) : add_edge!(g, Edge(j, i))
end
```

Reply:
Sure, I'll be willing to.

Reply:
Should also use `rand(Bool)`

Reply:
Oh, yeah, good catch. @shizejin, when you're testing a coin flip, it's better to use a boolean. That way the mechanics are taken care of for you:

```
rand(Bool) ? do_if_true() : do_if_false()
```

Reply:
Thanks! It's great to learn about this. Will modify this in minutes.